compiler/plugins/target/AMD-AIE/aie/test/via_DMA_test.mlir

```diff
@@ -1,6 +1,8 @@
 
 // RUN: iree-opt --amdaie-objectFifo-stateful-transform %s | FileCheck %s
 
+// XFAIL: *
```

Comment:
This test isn't supported anymore because of the `via_dma`? In that case, we could just remove the test?